spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/ConformanceExecution.scala

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.conformance
+
+import java.io.{PrintWriter, StringWriter}
+
+import org.apache.spark.sql
+import org.apache.spark.sql.functions.{lit, to_date}
+import org.apache.spark.sql.{DataFrame, Row, SparkSession}
+import za.co.absa.atum.AtumImplicits
+import za.co.absa.atum.AtumImplicits._
+import za.co.absa.enceladus.common.Constants.{InfoDateColumn, InfoDateColumnString, InfoVersionColumn, ReportDateFormat}
+import za.co.absa.enceladus.common.RecordIdGeneration.{getRecordIdGenerationStrategyFromConfig}
```

Comment:
```suggestion
import za.co.absa.enceladus.common.RecordIdGeneration.getRecordIdGenerationStrategyFromConfig
```

Reply:
And in general `getRecordIdGenerationStrategyFromConfig` could be `RecordIdGeneration.getStrategyFromConfig`. Would be more telling than a random method with long name popping up.

Reply:
Yea, Idea suggested and I didn't notice